test/unit/plugins/provisioners/ansible/cap/guest/pip/pip_test.rb

```diff
@@ -0,0 +1,52 @@
+require_relative "../../../../../../base"
+
+require Vagrant.source_root.join("plugins/provisioners/ansible/cap/guest/pip/pip")
+
+describe VagrantPlugins::Ansible::Cap::Guest::Pip do
+  include_context "unit"
+
+  subject { VagrantPlugins::Ansible::Cap::Guest::Pip }
+
+  let(:iso_env) do
+    # We have to create a Vagrantfile so there is a root path
+    env = isolated_environment
+    env.vagrantfile("")
+    env.create_vagrant_env
+  end
+
+  let(:machine) { iso_env.machine(iso_env.machine_names[0], :dummy) }
+  let(:communicator) { double("comm") }
+
+  before do
+    allow(machine).to receive(:communicate).and_return(communicator)
+    allow(communicator).to receive(:execute).and_return(true)
+  end
+
```

Comment:
Please fix the indentation within the `describe` blocks below. They appear to be indented with 4 spaces while 2 are used elsewhere.

Reply:
Should be fixed in most recent commit.